doc/lua_api.txt

```diff
@@ -4912,12 +4912,13 @@ Timing
 Server
 ------
 
-* `minetest.request_shutdown([message],[reconnect],[delay])`: request for
+* `minetest.request_shutdown([message],[reconnect],[delay],[countdown])`: request for
   server shutdown. Will display `message` to clients.
     * `reconnect` == true displays a reconnect button
     * `delay` adds an optional delay (in seconds) before shutdown.
       Negative delay cancels the current active shutdown.
       Zero delay triggers an immediate shutdown.
+    * `countdown` message displayed during countdown
```

Comment:
the name is misleading like this, should be renamed to e.g. `countdown_msg`

Reply:
renamed